access-mgmt/README.adoc

```diff
@@ -0,0 +1,473 @@
+:consumer-config-url: consumer-configuration/README.adoc
+:service-binding-url: service-discovery/README.adoc
+:access-mgmt-url: access-mgmt/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-access"]
+= Managing account access in {product-long}
+ifdef::context[:parent-context: {context}]
+:context: managing-access
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As an organization administrator in {product}, you can manage the level of access for user accounts and service accounts in Kafka instances. You can allow or deny operations that specific accounts or that all accounts in your organization can perform for an instance or for certain resources in that instance, such as topics. You can also allow or deny access for other users or service accounts to define account permissions for an instance.
+
+If you're not an administrator in {product}, your own user account has full access to the Kafka instances that you create, and you can manage the level of access for service accounts that you create for your own instances. However, you can't manage the level of access of other users or of service accounts created by other users for your Kafka instances or for any other Kafka instances.
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
+
+[id="con-access-management_{context}"]
+== Access management in {product}
+
+[role="_abstract"]
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable an organization administrator to manage how user accounts and service accounts are permitted to interact with Kafka resources, such as instances and topics. A user account enables users in your organization to access resources. A service account enables your application or tool to connect securely to your resources.
+
+Administrators can allow or deny operations that specific accounts or that all accounts in the organization can perform for Kafka resources. A resource in an ACL can be a Kafka instance, topic, consumer group, or producer transaction. Administrators can also allow or deny access for other users to define account permissions for an instance.
+
+Users who aren't administrators can fully access a Kafka instance that they create and can manage the level of access for service accounts that they create for their own instances. However, only administrators can manage the level of access of other users or of service accounts created by other users for a Kafka instance.
```

Comment:
Same comment as above. An instance owner has full control to manage access for any user or service account in the organization.

Reply:
Fixed similar to above, to this:

>{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. A resource in an ACL can be a Kafka instance, topic, consumer group, or producer transaction. A user account enables users in your organization to access your resources. A service account enables your application or tool to connect securely to your instance and access your resources.